<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadImageWgtClass</class>
 <widget class="QWidget" name="LoadImageWgtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1221</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoadImageWgt</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>图像导入结果</string>
     </property>
     <layout class="QHBoxLayout" name="HLayout_ImagePainter"/>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>240</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btn_selLoadImg">
        <property name="text">
         <string>选择导入文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_selFileTag_2">
        <property name="title">
         <string>文件标签</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_Img_FileName">
           <property name="text">
            <string>文件名：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_file_Path">
           <property name="text">
            <string>文件路径：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_Img_FileType">
           <property name="text">
            <string>文件类型：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_img_LoadState">
           <property name="text">
            <string>加载状态：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_img_Size">
           <property name="text">
            <string>图像尺寸：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_Img_CrDepth">
           <property name="text">
            <string>颜色深度：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_Img_CrFormat">
           <property name="text">
            <string>颜色格式：</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>导入尺寸</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>最大宽度</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="SBox_LoadPicWidth">
             <property name="maximum">
              <number>4080</number>
             </property>
             <property name="value">
              <number>1920</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>最大高度</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="SBox_LoadPicHeight">
             <property name="maximum">
              <number>4080</number>
             </property>
             <property name="value">
              <number>1080</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_autoShrink">
        <property name="text">
         <string>自动尺寸转化</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_shrinkImg">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>尺寸转化</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_autoSaveDataCenter">
        <property name="text">
         <string>自动存入数据中心</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_saveImgToDataCenter">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>存入数据中心</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_CrMetric">
        <property name="text">
         <string>颜色度量空间自动导入图片</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
